CHANGELOG.rst

```diff
@@ -8,11 +8,14 @@ Unreleased
 
 - Validate that the supplied source directory is valid (`PR #260`_, Fixes `#259`_)
 - Set and test minimum versions of build's runtime dependencies (`PR #267`_, Fixes `#263`_)
+- Use symlinks when creating venv's on where available (`PR #274`_, Fixes `#271`_)
```

Comment:
Maybe change to `Use symlinks when creating venv's when available`?

Reply:
Sure, a tiny bit better and the current version has an extra word anyway. Any thoughts to the PyPy3 test and the xfail? (Possibly more for @gaborbernat, who seems to have looked into it 6 months ago)

Reply:
Not really, sorry. I haven't looked at that in a while. I'll defer to @gaborbernat as he was the one who introduced this in the first place and may have more context than me right now.